### ISSUE TITLE
Bug: Scrolling catches in hidden Map element

### ISSUE THREAD

**Issue by Yogibuff** [Yogibuff/SnowTracks#15]
Scroll function prevented when cursor is within the map.
- The Google Map is causing this, and it is present on the index.html and about.html pages
- The file that will be changed: initMap.js
- The area affected is 400px tall, 50% of the viewport wide. 
- This scrolling issue occurs whether the map is visible or not.

This bug made me take a look at the scroll function of the Google Map, and decide if it was affecting the user experience. Although scrolling is a useful while the map is visible, I plan on disabling this "scrollwheel" functionality.

TLDR conclusion: Remove 'scrollwheel' functionality within SnowTracks' embedded Google Maps

**Comment by Yogibuff** [comment]
Additional notes on the issue:
- The area where the scroll is affected by the embedded map is 400px tall, 50% of the viewport wide.
- Scrolling comes default for Google map, but there are alternatives like double clicking and buttons.
- Efficiently using the mouse-wheel within the map area can be awkward for users
- Allowing the 'scrollwheel' function causes scrolling to appear non-responsive when the map is hidden while the mouse is within the map area.

There is an issue where scrolling appears to be non-responsive when the map is hidden, and it can really confuse the user. This issue occurs when a user attempts to scroll down the page with their mouse wheel while they have their cursor in the map's location - a 400px by 50% wide rectangle in the center of the screen).

Allowing the user to have this zoom functionality when using the maps was not substantively contributing to a feature, the user experience, and the ability to use the map. Therefore, maps embedded in SnowTracks now require double clicking or using the zoom buttons (map bottom right corner).

**Comment by Yogibuff** [comment]
This bug also contextually relates to issue #13, which also employs links to Google Maps. 

I will make these resort links open google.com/maps in a new tab/window, <em>with scrolling enabled.</em> 

Scrolling will be <em>disabled</em> on maps within SnowTracks, so that users can use their mouse wheel to navigate the site without the hidden map listening to the scrollwheel.
